packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2700,7 +2701,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             // Please note that this does not change file format, so it can be disabled in the future if this
             // optimization no longer makes sense (for example, batch compression may make it less appealing).
             if (this.currentlyBatching() && type === ContainerMessageType.Attach &&
-                    this.mc.config.getBoolean("Fluid.ContainerRuntime.disableAttachOpReorder") !== true) {
+                this.mc.config.getBoolean("Fluid.ContainerRuntime.disableAttachOpReorder") !== true) {
```

Comment:
@tylerbutler - Here's the problem.  Please cherry-pick (or manually apply) this commit https://github.com/microsoft/FluidFramework/pull/12062/commits/a2eb5e0b7f69a28f345beb130d03eaea78c2abfd